node/bft/ledger-service/src/prover.rs

```diff
@@ -56,6 +56,11 @@ impl<N: Network> LedgerService<N> for ProverLedgerService<N> {
         unreachable!("Latest block does not exist in prover")
     }
 
+    /// Returns the latest restrictions ID in the ledger.
+    fn latest_restrictions_id(&self) -> Field<N> {
+        Field::zero()
+    }
+
```

Comment:
TBH I feel like it should be considered here as well. Rather than return useless unused data this should be an Option and prover's should return `None`